tmk_core/common/mousekey.c

```diff
@@ -288,13 +280,158 @@ void mousekey_off(uint8_t code) {
 #endif
 }
 
+#elif defined MK_ACCEL2
 
+#define CLAMP(x, low, high)  (((x) > (high)) ? (high) : (((x) < (low)) ? (low) : (x)))
+#ifndef MOUSEKEY_SLOW_RATIO
+#define MOUSEKEY_SLOW_RATIO 4
+#endif
 
+uint8_t mk_delay = MOUSEKEY_DELAY/10;
+uint8_t mk_interval = MOUSEKEY_INTERVAL;
+uint8_t mk_max_speed = MOUSEKEY_MAX_SPEED;
+uint8_t mk_slow_speed = MOUSEKEY_MAX_SPEED / MOUSEKEY_SLOW_RATIO;
+uint8_t mk_time_to_max = MOUSEKEY_TIME_TO_MAX;
+uint8_t acc_per_int = ((uint16_t) MOUSEKEY_MAX_SPEED) * MOUSEKEY_INTERVAL / MOUSEKEY_TIME_TO_MAX;
+uint8_t mk_wheel_max_speed = MOUSEKEY_WHEEL_MAX_SPEED;
+uint8_t mk_wheel_time_to_max = MOUSEKEY_WHEEL_TIME_TO_MAX;
+
+// x positive right, y positive down
```

Comment:
these are magic numbers. Please turn them into defines

Reply:
I'm just following the current code.
https://github.com/qmk/qmk_firmware/blob/3a7c65b42442971bcb3369cc9dc3615b6aad1383/tmk_core/common/mousekey.c#L53-L64
https://github.com/qmk/qmk_firmware/blob/3a7c65b42442971bcb3369cc9dc3615b6aad1383/tmk_core/common/mousekey.c#L126-L130

Reply:
I was mostly concerned with the directions. I like the change you made. Much clearer to follow.